### ISSUE TITLE
Fenced code blocks can't be syntax highlighted in Atom

### ISSUE THREAD

**Issue by jkrumbiegel** [JuliaDocs/Documenter.jl#1163]
I don't think this qualifies as an Atom issue, because it's related to Documenter's syntax.

If I have a fenced block like this, Atom shows the julia code with correct syntax highlighting:

```julia
``julia
a = 1 + 2
``
```

If I turn this into a Documenter `@example` block, I can't use julia syntax highlighting, because it seems the block is only identified correctly like this, where Atom can't know it's julia code:

```julia
``@example
a = 1 + 2
``
```

That makes it hard to correctly edit the code, because everything is colored as one string.
This syntax doesn't work, the block is not registered as an `@example`:

```julia
``julia
@example
a = 1 + 2
``
```

In my opinion, Documenter should respond to `@whatevertheblockis` even if this comes only in the first line after the `julia` declaration.

(In all examples I've replaced the triple backtick with two for syntax highlighting to work here)

**Comment by fredrikekre** [comment]
Are you suggesting that we change Documenter syntax from
````
```@example
a = 1 + 2
```
````
to
````
```julia
@example
a = 1 + 2
```
````
? 

Seems like it would be much easier to teach Atom about these blocks and color them as julia syntax.

**Comment by jkrumbiegel** [comment]
I don't think that the syntax would need to be changed, I would just add a rule that  a `julia` right after the triple backtick is ignored and that the block identifiers can be used after it. In my understanding the first word after triple backticks is used almost everywhere as an optional descriptor for the programming language that is fenced, so this would just be in line with the usual convention. Old code wouldn't break either, unless the first line happens to start immediately with the word julia

**Comment by mortenpi** [comment]
If the user has e.g. an `@example` macro, then they could quite easily run into a case where their normal code examples get interpreted as at-blocks, no?

I'm with @fredrikekre here -- I think updating the Markdown highlighter for Atom would be the better way to go. Any ideas if it is possible to have "extension packages" for [language-markdown](https://atom.io/packages/language-markdown)?

**Comment by jkrumbiegel** [comment]
Interesting, I was about to post this issue at Juno.jl, but I wanted to screenshot an example. I had recently installed the Weave.jl plugin for Atom and in the list of available syntax highlightings there was now a Weave.jl markdown option. This option does exactly what I want, it highlights julia syntax correctly when the block starts with something like `@example` or `@eval`. It's clearly aware of these words because `@exampl` doesn't work. So, I'm just posting this here for others who might look for this functionality, you would have to manually select the correct formatting because .md files get Github markdown highlighting by default.

**Comment by odow** [comment]
Closing because this is better handled by editors, not Documenter, and as @jkrumbiegel there are options for Atom.
